services/auth/src/api.js

```diff
@@ -376,6 +386,35 @@ builder.declare({
 
   return res.reply({ auditHistory: rows.map(c => auditToJson(c)), continuationToken });
 });
+
+builder.declare({
+  method: 'get',
+  route: '/audit-logs/:clientId',
```

Comment:
this route is a bit inconsistent with existing one (audit vs audit-logs)
```
  route: '/audit/:entityType/:entityId',
```

Plus, maybe it should be defined as a client-specific route:
```
route: '/clients/:clientId/audit',
```